pkgs/applications/networking/filebrowser/default.nix

```diff
@@ -0,0 +1,55 @@
+{ buildGoModule, buildNpmPackage, fetchFromGitHub, lib }:
+
+let
+  frontend = buildNpmPackage rec {
+    pname = "filebrowser-frontend";
+    version = "2.23.0";
+
+    src = fetchFromGitHub {
+        owner = "filebrowser";
+        repo = "filebrowser";
+        rev = "v${version}";
+        sha256 = "sha256-xhBIJcEtxDdMXSgQtLAV0UWzPtrvKEil0WV76K5ycBc=";
+    } + "/frontend";
+
+    npmDepsHash = "sha256-acNIMKHc4q7eiFLPBtKZBNweEsrt+//0VR6dqwXHTvA=";
+
+    NODE_OPTIONS = "--openssl-legacy-provider";
```

Comment:
Hm, why is it needed? We have EOLed Node.js 16 which I believe is the last affected by this issue now.

Reply:
it's using Node.js v18.16.0, and I don't know why, I only know my build fails without it:

```
    at FSReqCallback.oncomplete (node:fs:198:23) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v18.16.0

ERROR: `npm build` failed
```

Reply:
can you bump it to node 20?

Reply:
I bumped it with:

```
  frontend = buildNpmPackage.override { nodejs = nodejs_20; } rec {
    pname = "filebrowser-frontend";
```

But the error remains the same:

```
    at FSReqCallback.oncomplete (node:fs:188:23) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v20.2.0

ERROR: `npm build` failed
```